image_width: 640
image_height: 360
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [553.627733, 0.000000, 317.448667, 0.000000, 550.558377, 122.189254, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.519086, 0.331704, 0.013667, 0.002975, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [457.295868, 0.000000, 319.854539, 0.000000, 0.000000, 514.913330, 118.546649, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
